coalib/testing/LocalBearTestHelper.py

```diff
@@ -193,6 +193,9 @@ def check_results(self,
         :param lines:            The lines to check. (List of strings)
         :param results:          The expected list of results.
         :param filename:         The filename, if it matters.
+        :param check_order:      Whether to check that the elements of results
+                                 (expected list) and that of the list generated
+                                 from get_results are in the same order or not.
```

Comment:
That `get_results` is used is an implementation detail and I wouldn't mention it explicitly.